L10nSnapshotTests/L10nMktSuiteSnapshotTests.swift

```diff
@@ -0,0 +1,164 @@
+        app.switches["SystemThemeSwitchValue"].tap()
+    
+        app.cells.staticTexts.element(boundBy: 6).tap()
+        navigator.goto(HomePanelsScreen)
+        snapshot("DefaultTopSites-01")
+    }
+    
+    func test4PrivateBrowsingTabsEmptyState() {
+        navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
+        app.buttons["closeTabButtonTabTray"].tap()
+        waitForExistence(app.tables["Empty list"], timeout: 3)
+        snapshot("PrivateBrowsingMode")
+    }
+    
+    func test5DefaultSearchEngine() {
+        navigator.goto(SearchSettings)
+        XCTAssert(app.tables.staticTexts["Google"].exists)
+        snapshot("SearchSuggestions")
+        
+        // Disable Dark Mode
+        navigator.goto(SettingsScreen)
+        navigator.goto(DisplaySettings)
+        app.switches["SystemThemeSwitchValue"].tap()
+    }
+    
+    func testSearchWidgets2() {
+        // Set a url in the pasteboard
+        let springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
+        navigator.openURL("pocket.com")
+        waitUntilPageLoad()
+        navigator.performAction(Action.PinToTopSitesPAM)
+
+        // Navigate to topsites to verify that the site has been pinned
+        navigator.nowAt(BrowserTab)
+        navigator.openNewURL(urlString: "mozilla.org")
+        waitUntilPageLoad()
+        navigator.performAction(Action.PinToTopSitesPAM)
+        
+        setupSnapshot(springboard)
+        // Open the app and set it to background
+        app.activate()
+        sleep(1)
+        snapshot("DefaultBrowser-01")
+        sleep(1)
+        XCUIDevice.shared.press(.home)
+        
+        // Swipe Right to go to Widgets view
+        let window = springboard.children(matching: .window).element(boundBy: 0)
+        window.swipeRight()
+        window.swipeRight()
+        // Swipe Up to get to the Edit and Add Widget buttons
+        // This line is needed the first time widgets view is open
+        springboard.alerts.firstMatch.scrollViews.otherElements.buttons.element(boundBy: 0).tap()
+        
+        let element = springboard/*@START_MENU_TOKEN@*/.scrollViews["left-of-home-scroll-view"]/*[[".otherElements[\"Home screen icons\"].scrollViews[\"left-of-home-scroll-view\"]",".scrollViews[\"left-of-home-scroll-view\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.children(matching: .other).element.children(matching: .other).element(boundBy: 0)
+            element.swipeUp()
+            element.swipeUp()
+            element.swipeUp()
+            springboard.scrollViews["left-of-home-scroll-view"].otherElements.buttons.firstMatch.tap()
+
+        sleep(1)
+        springboard.otherElements["Home screen icons"].buttons.firstMatch.tap()
+
+        // Select Fennec (username)
+        springboard.collectionViews.cells["Fennec (synctesting)"].children(matching: .other).element.children(matching: .other).element(boundBy: 0).children(matching: .other).element.tap()
```

Comment:
Is this still synctesting?

Reply:
Since I was running latest tests on that laptop (fastlane does not seem to work on Big Sur) I added that that way but as commented above, depending on where you launch these tests to get the screenshots, that has to be updated.